lib/test_case_manager_filter.rb

```diff
@@ -40,6 +40,8 @@ def done
       end
 
       config.on_event :test_case_finished do |event|
+        puts "test_case_finished: #{event}"
+        puts "#{event.inspect}\n"
```

Comment:
It will looks like,
```
07-05 14:30:08.516  test_case_finished: #<Cucumber::Events::TestCaseFinished:0x00000000101bf890>
07-05 14:30:08.516  #<Cucumber::Events::TestCaseFinished:0x00000000101bf890 @test_case=#<Cucumber::Core::Test::Case: features/tierN/web/admin-console/search.feature:14>, @result=#<Cucumber::Core::Test::Result::Passed:0x00000000101bfb88 @duration=#<Cucumber::Core::Test::Result::Duration:0x00000000101bfc00 @nanoseconds=207136595458>>>
```
I'd like to check the output in more situations from CI.